src/Microsoft.Health.Dicom.Core/Features/Common/FileProperties.cs

```diff
@@ -8,4 +8,8 @@ namespace Microsoft.Health.Dicom.Core.Features.Common;
 public class FileProperties
 {
     public long ContentLength { get; init; }
+
+    public string ETag { get; init; }
```

Comment:
I was thinking that we could perhaps rely on the `Azure.Core` library's `ETag` type, but that may be a mistake to have all of our libraries take a dependency on that lib just for the type